example/90-shoot.yaml

```diff
@@ -105,6 +105,12 @@ spec:
     # zones: # optional, only relevant if the provider supports availability zones
     # - europe-central-1a
     # - europe-central-1b
+    # settings: # optional, it allows to configure the machine-controller.
```

Comment:
Wouldn't `machineControllerManager` or `managementSettings` or similar be better here? `settings` alone is quite unspecific and vague (most other fields are "settings" for the worker pool itself, but this section here is about the mgmt of them)

Reply:
Hm, agree with the reasoning. I'll change it to `machineControllerManager` then.

Reply:
Though wondering if it would be end-user friendly, open for other suggestions as well meanwhile :)

Reply:
I have renamed it to `machineControllerManager`, and the struct to `MachineControllerManagerSettings` just to be clearer. ref: https://github.com/gardener/gardener/commit/9aacdeb0348a4d06173bb54d71af652807da0846#diff-9a667891729b65e0ebbf3feef920fcd9R761